src/control/server/mgmt_pool_test.go

```diff
@@ -34,6 +34,23 @@ import (
 	"github.com/daos-stack/daos/src/control/system/raft"
 )
 
+var (
+	mockTierBytes   = []uint64{uint64(1), uint64(2)}
```

Comment:
minor - I don't think these need to be casted within the array

Reply:
will fix if I have to repush